### PR TITLE
Reduce package size

### PR DESCRIPTION
**Pull request by nechaido**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,12 @@
   "browser": {
     "./metasync.js": "./dist/metasync.js"
   },
+  "files": [
+    ".babelrc",
+    "dist/",
```
Comment (tshemsedinov) [inline package.json]: Do we need both published (`dist` and `lib`)? Can't we build `dist` after install?

Comment (nechaido) [reply package.json]: AFAIK we would need to make `babel` our runtime dependency.

File: package.json [hunk 2]
```diff
@@ -33,6 +33,12 @@
...
+    "lib/",
+    "metasync.js"
```
Comment (belochub) [inline package.json]: It's better to remove this and fix the `main` field to point correctly to `metasync.js`, not `./metasync.js`.
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -33,6 +33,12 @@
   "browser": {
     "./metasync.js": "./dist/metasync.js"
   },
+  "files": [
+    ".babelrc",
```
Comment (belochub) [inline package.json]: Also, I don't think there is any point in including `.babelrc` in the package.

**Pull request merged**